### PR TITLE
#565 - Teloscan homepage - Global Layout

### LINKED ISSUE #565: Teloscan Homepage Redesign - global layout
## Description

After merging many new PRs with components developed concurrently, we had different layouts on each new page. This issue aims to standardize a unified layout to follow on all pages, along with some minimal style adjustments.

## Acceptance Criteria
- All pages should have the Title at the same level
- All pages showing any kind of container component like a card should align with the site header (width)
- The footer should be separated from any content above.
- The footer should never be separated from the bottom.
- The Transaction Table should not show horizontal scroll on Desktop resolutions

image

image

image

image

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #565

I closed this PR since it's included in this other PR: https://github.com/telosnetwork/teloscan/pull/576

https://deploy-preview-566--dev-mainnet-teloscan.netlify.app/

## Test scenarios
Go throw this pages and check all the points listed in the issue are solved:
- https://deploy-preview-566--dev-mainnet-teloscan.netlify.app/txs
- https://deploy-preview-566--dev-mainnet-teloscan.netlify.app/block/331138389
- https://deploy-preview-566--dev-mainnet-teloscan.netlify.app/tx/0x098b50d3a338b694d9c6b108e42c6c4dcb0518bb4072bbcba8e5362765db7c44

## Screenshots

image
image
image
image
image

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all english text to the translation file and/or created a new issue with the required translations for the currently supported languages
-   [x] I have tested for mobile functionality and responsiveness
-   [x] I have added appropriate test coverage

### CODE REVIEW

File: src/lib/utils.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ export function formatWei(bn, tokenDecimals, displayDecimals) {
     const formatted = ethers.utils.formatUnits(amount.toString(), (tokenDecimals || WEI_PRECISION));
     let str = formatted.toString();
     // Use string, do not convert to number so we never loose precision
+
+
+    // tenemos que agregar una cantidad de ceros igual a displayDecimals
```
Comment (donnyquixotic) [inline src/lib/utils.js:41]: comentarios en inglés por favor

File: src/css/global/_global.scss
Comment (donnyquixotic) [inline src/css/global/_global.scss:1]: empty file?

**Review by donnyquixotic — CHANGES_REQUESTED** [review]
Great Job! minor feedback

File: src/lib/utils.js (hunk 1 above)
Comment (Viterbo) [reply src/lib/utils.js:41]: nuuuuuuuuu :see_no_evil:

**Review by donnyquixotic — APPROVED** [review]

**Pull request closed without merging**